### ISSUE TITLE
Docs - Rewrite Getting Started

### ISSUE THREAD

**Issue by cheeseplus** [test-kitchen/test-kitchen#1456]
## Description

The current Getting Started guide is 17 pages which is decidedly too long for an intro. With some upcoming structural changes as part of a Hugo migration this will be easier to catalogue idiosyncracies in their own reference section.

Additionally git-daemon is a terrible example and it's been unwieldy so let's use something else.

**Comment by PurityControl** [comment]
After just having visited the site and read the getting started section, I was trying to find a way to get in touch with the authors and congratulate them and came across this issue.

Maybe the git_daemon example could be improved upon, but I have to give a big congratulations to the quality of the documentation.

I have a ruby background but I was able to become familiar with kitchen and get a very good grasp of how everything tied together in under an hour by following the tutorial. It was clear and lucid without being too verbose.

I then walked through the tutorial with a colleague who is just starting out in the world of software development and has no provisioning experience. They also praised the quality of documentation and were very appreciative of how helpful it was for people who may not be provisioning or software development experts.

Things can always be improved but I just wanted to reach out and say a very well done. This seemed as good a place as any to pass that message on.
